linux/evolution.md

```diff
@@ -1 +1,49 @@
-# Linux - Evolution
\ No newline at end of file
+# Linux - Evolution
+
+
+## History and Evolution
+
+Linux Kernel main differences among the first commit and the latest one, is that the first it's main purpose is
+for testing and no other "commercial" use, unlike the current version, which is available and stable. 29 Years after the first release, other changes are, for the 0.11 version the system must have at least 4mb of RAM, a 32 bit 
+computer. On the Software side, the newer version can manage multiple filesystem types, like ntfs, ext, btrfs, as opposed to the first one, which only supported ext filesystem. This one also supported only 3 keyboard. In summary, 
+the original spirit of the kernel remains the same, and it has been boosted with new technology, compatibility, and numerous new functions, among other things.
+
+## Evolution Chart
+
+![alt text](assets/Files-vs-Version.png "Image Example")
+
+## Architecture Decision Records
+
+1. **Title:** ADR 1: Greybus Support for version 4.9
+    1. **Context:** Greybus is intended for mobile modularity leaded by Project Ara. Google cancelled the project. 
+    The Greybus support seems useless without Project Ara.
+    1. **Decision:** Continue development on Greybus support.
+    1. **Status:** Implemented, and added to the version 4.9-5.10-rc1-HEAD
+    1. **Consequences:** Used in Android smartphones.
+
+1. **Title:** ADR 2: Ext4 Support 2.6
+    1. **Context:** New filesystem type updated ext4.
```

Comment:
The context and decision are not clear.. Is it about adding for the first time support for the ext4 filesystem? Why it was added? What is the 2.6 described in the name?
What are the usage consequences of including the filesystem?